forge-gui/res/cardsfolder/upcoming/arthur_marigold_knight.txt

```diff
@@ -5,7 +5,7 @@ PT:4/5
 K:Haste
 T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | IsPresent$ Creature.attacking+Other | NoResolvingCheck$ True | Execute$ TrigDig | TriggerDescription$ Whenever CARDNAME and at least one other creature attack, look at the top six cards of your library. You may put a creature card from among them onto the battlefield tapped and attacking. Put the rest on the bottom of your library in a random order. Return that creature to its owner's hand at end of combat.
 SVar:TrigDig:DB$ Dig | DigNum$ 6 | ChangeNum$ 1 | Optional$ True | Reveal$ True | ChangeValid$ Creature | DestinationZone$ Battlefield | Tapped$ True | Attacking$ True | RestRandomOrder$ True | RemembedChanged$ True | SubAbility$ DBDelayedTrigger

```

Comment:
you've just cleaned up redundant code

but I'd say the "Remembed" typo is responsible :p